packages/web/src/builders/package/package.impl.ts

```diff
@@ -95,15 +93,15 @@ export function run(
             localResolve(),
             resolve({ extensions: fileExtensions }),
             babel({
-              ...createBabelConfig(options, options.projectRoot, esm),
+              ...createBabelConfig(options, options.projectRoot),
               extensions: fileExtensions,
               externalHelpers: false,
               exclude: 'node_modules/**'
             }),
             commonjs(),
             filesize()
           ];
-          if (esm) {
+          if (index === 0) {
             // TS plugin has to come first before types are stripped, otherwise
             plugins.unshift(
               typescript({
```

Comment:
babel runs typescript anyway - the only reason we really need this is to generate definition files
so we only run the **once**